docs/building-blocks.md

```diff
@@ -297,7 +297,7 @@ end
 SfnRegistry.register(:instance_size_default){ 'm3.medium' }
 ~~~
 
-With the value registered, it can then be referenced:
+With the value registered, holding an array of values, it can then be referenced:
```

Comment:
This reads oddly to me. Maybe: "With the array registered as `instance_size`, we can now reference it:"